package.json

```diff
@@ -55,7 +55,6 @@
   "devDependencies": {
     "dat-node": "^3.5.0",
     "marked": "^0.3.6",
-    "nodemon": "^1.10.0",
```

Comment:
we'd like to keep this in devDependencies b/c its pretty big dependency. Why do you want to move it?